fhir-server-test/src/test/java/com/ibm/fhir/server/test/ExampleRequestProcessor.java

```diff
@@ -1,4 +1,4 @@
-/*
+/**
```

Comment:
this is non-javadoc so the single `*` is correct / consistent with the rest of our files